### PR TITLE
Add hostname to triggerer metric [triggers.running]

### PR DESCRIPTION
**Pull request by apilaskowski**
It is impossible to distinguish informations from different triggerers.
We can not approximate the number of triggers that are running at given moment.
Giving hostname suffix will make it possible to approximate this information.

### CODE REVIEW

**Comment by potiuk** [comment]
Errors. Also maybe @ferruzzi Might have a say after you fix them

File: airflow/jobs/triggerer_job_runner.py [hunk 1]
```diff
@@ -401,7 +401,7 @@ def handle_failed_triggers(self):
             Stats.incr("triggers.failed")
 
     def emit_metrics(self):
-        Stats.gauge("triggers.running", len(self.trigger_runner.triggers))
+        Stats.gauge(f"triggers.running.{self.job.hostname}", len(self.trigger_runner.triggers))
```
Comment (ferruzzi) [inline airflow/jobs/triggerer_job_runner.py:404]:
```suggestion
        Stats.gauge(f"triggers.running.{self.job.hostname}", len(self.trigger_runner.triggers))
        Stats.gauge(f"triggers.running", len(self.trigger_runner.triggers), tags={"hostname":self.job.hostname})
```

For now we are going to have to double-emit metrics which embed values like that due to OpenTelemetry's rather short max name length, but it supports tagging to make up for it.

**Review by ferruzzi — CHANGES_REQUESTED** [review]
Thanks for the tag, Jarek.

Two requests:

1. The Metrics documentation page is not yet auto-generated, so please update the name there as well (found [here](https://github.com/apache/airflow/blob/main/docs/apache-airflow/administration-and-deployment/logging-monitoring/metrics.rst)).  I know it's a little out of date on other names as well, we're working on it.
2. Please see the suggested change regarding tagging

**Comment by apilaskowski** [comment]
I will update the doc asap (tomorrow morning it is 8pm in Poland right now). Thank you for explaining why we need those tags!

**Comment by ferruzzi** [comment]
I also just noticed I missed a space in the tags dict in my suggestion code, you'll want to run static checks on it and it'll fix that.

**Review by uranusjr — APPROVED** [review]

**Comment by apilaskowski** [comment]
Are those changes responsible for Core Providers (amazon) problems? @ferruzzi @kosteev

**Comment by apilaskowski** [comment]
@potiuk @ferruzzi
I updated entry for this metric in documentations, is it correct now?

**Review by ferruzzi — APPROVED** [review]
I don't have a full understanding of Triggerers yet, so I wanted to double check I wasn't missing something there. :+1: 

Unless someone chimes in otherwise on that point, I'm happy with it.

((@vandonr-amz  - Maybe a quick gut check on this?))

**Review by vandonr-amz — APPROVED** [review]
LGTM

**Review by potiuk — APPROVED** [review]

**Pull request merged**